index.bs

```diff
@@ -592,7 +592,6 @@ A [=sensor=] has an associated <dfn>current polling frequency</dfn> which is ini
 <pre class="idl">
 [SecureContext]
 interface Sensor : EventTarget {
-  readonly attribute SensorState state;
```

Comment:
In origin trials the API call statistics is collected, so IMO it is preferable to leave something and see whether is is used or not. I would personally leave the `activated` attribute: it is minimalistic and perfectly aligned with the rest of API. Exposing both is also possible, so I'm fine with any decision regarding this  :)